### ISSUE TITLE
Transfer name domain ownership for obofoundry.org from Dotster to GO AWS / Route 53

### ISSUE THREAD

**Issue by cmungall** [OBOFoundry/purl.obolibrary.org#393]
Currently the registrant is barry, admin/tech is lbl

we will transfer to google domains, and this will be co-administered across obo

**Comment by kltm** [comment]
To move forward on this, the eventual registrar needs to initiate the transfer. Whoever is controlling where we want it to be should log in and start the process, and we'll try and keep an eye on it to make sure it can get released at this end.

**Comment by cmungall** [comment]
@bpeters42 needs to initiate the transfer

**Comment by kltm** [comment]
Also see: https://github.com/OBOFoundry/OBOFoundry.github.io/issues/29

**Comment by cmungall** [comment]
@bpeters42 any update on this?

**Comment by cmungall** [comment]
@kltm I believe the name is transferred?

**Comment by kltm** [comment]
It has not. There is still a conversation with LBL to be had as it seems that while the ownership is with a commercial provider, all administration is LBL. This means that the xfer needs to be a more coordinated effort.

**Comment by kltm** [comment]
Just nudging a little life into this. 
The EC2 instance that this is currently on will be EoLed on August 22nd, 2022. This is about a year away. Ideally, once we have the Route 53 contol of this record, we could experiment a little better with getting rid of this machine.

**Comment by kltm** [comment]
This is now also part of https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1674

**Comment by kltm** [comment]
Tl;DR: In summary, we are already in functional control, besides ownership; this was accomplished late February 2020. I'm not sure why it slid off the radar after that, but we remembered enough to use this to setup the dashboard subdomain.

Okay, after some digging, here is a review of the current state of affairs. 

* The domain is held in Dotster; we have access and the domain seems prepped to be transferable
* The domain will renew this February, with a March expiration; I suggest that we use this as a deadline for any transfer
* The domain has rather complicated DNS records associated with it, but I have reason to believe that they are (largely) not respected
* The DNS NS for the domain is setup to use AWS name servers
* We seem to be controlling domain records _right now_ through a Route 53 Hosted Zone 
* In spite of how the contacts are listed, as well as some email threads, I believe that LBL no longer has any connection to the domain

**Comment by kltm** [comment]
@cmungall Moving forward, I would suggest a refresh email to let the owner know that myself and @abessiari (or anybody else) may be doing some work in the account to try and finalize the transfer before February.

**Comment by nlharris** [comment]
Is there anything else that needs to be done here?

**Comment by abessiari** [comment]
Yes I am working on it.  I am moving  the domain obofoundry.org  from Dotser to AWS. 
If all goes well it should be done today.

**Comment by kltm** [comment]
We currently seems to have lost access to Dotster. Looking into this. Has there been a credential change? @cmungall @jamesaoverton

**Comment by jamesaoverton** [comment]
I haven’t had anything to do with Dotster.

**Comment by kltm** [comment]
@cmungall @jamesaoverton This has now been completed. This is under the LBL AWS account.
